src/app/tab3/forgot-password/forgot-password.component.html

```diff
@@ -21,11 +21,15 @@
 
       <ion-card-content>
         <form [formGroup]="ForgetPasswordForm" (ngSubmit)="onSubmit()">
-        <div class="h-px-75">
+        <div class="h-px-90">
           <ion-item>
             <ion-label position="floating">Email</ion-label>
             <ion-input formControlName="username" type="text" placeholder="Enter email"></ion-input>
           </ion-item>
+          <ion-text color="danger" class="f14"
+          *ngIf="ForgetPasswordForm.get('username').invalid && ForgetPasswordForm.get('username').touched">
```

Comment:
No need to do all this we have a validity method in the base class and that will take care of this. Please use that Utility.